python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S6911.json

```diff
@@ -0,0 +1,21 @@
+{
+  "title": "\"tf.function\"s should not depend on global or free Python variables",
```

Comment:
Before merging the RSPEC and the rule, I think we should rephrase the title to avoid the awkward pluralization of `tf.function`, consistently with the other rules.